v1/metrics.yaml

```diff
@@ -429,115 +439,860 @@ paths:
+          in: path
+          description: If you want to filter by page-type, use one of content (edits on pages in content namespaces) or non-content (edits on pages in non-content namespaces). If you are interested in edits regardless of their page-type, use all.
+          type: string
+          enum: ['all', 'content', 'non-content']
+          required: true
+        - name: granularity
+          in: path
+          description: The time unit for the response data. As of today, supported values are daily and monthly
+          type: string
+          enum: ['daily', 'monthly']
+          required: true
+        - name: start
+          in: path
+          description: The date of the first day to include, in YYYYMMDD format
+          type: string
+          required: true
+        - name: end
+          in: path
+          description: The date of the last day to include, in YYYYMMDD format
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The list of values
+          schema:
+            $ref: '#/definitions/edits'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-route-filters:
+        - type: default
+          name: ratelimit_route
+          options:
+            limits:
+              internal: 100
+              external: 100
+      x-request-handler:
+        - get_from_backend:
+            request:
+              uri: '{{options.host}}/edits/aggregate/{project}/{editor-type}/{page-type}/{granularity}/{start}/{end}'
+              headers:
+                x-client-ip: '{{x-client-ip}}'
+      x-monitor: false
+
+  /edits/per-page/{project}/{page-id}/{editor-type}/{granularity}/{start}/{end}:
+    get:
+      tags:
+        - Analytics Edits Metrics API
+      summary: Get edit counts for a page in a project.
+      description: |
+        Given a Mediawiki project, a page-id and a date range, returns a timeserie of edit counts. You can filter by editors-type (all, anonymous, bot, registered). You can choose between daily and monthly granularity as well.
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: project
+          in: path
+          description: The name of any Wikimedia project formatted like {language code}.{project name}, for example en.wikipedia. You may pass en.wikipedia.org and the .org will be stripped off.  For projects like commons without language codes, use commons.wikimedia.  For projects like www.mediawiki.org, you can use that full string, or just use mediawiki or mediawiki.org. If you're interested in the aggregation of all projects, use all.
+          type: string
+          required: true
+        - name: page-id
+          in: path
+          description: The page-id to request edits for.
+          type: integer
+          format: int64
```

Comment:
Actually this is not true, page_id is an `int(10)` in the DB schema which is int32

Reply:
Noted, thanks !